apps-rendering/src/components/Byline/LiveBlogByline.tsx

```diff
@@ -0,0 +1,70 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { text } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { from, headline, neutral } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { darkModeCss } from 'styles';
+import { getThemeStyles } from 'themeStyles';
+import { DefaultByline } from './Byline.defaults';
+
+export const blogColor = (
```

Comment:
Given the `defaults` pattern you're using, would it make sense for this to live there since it's used in a couple of sub-components?